runconfig/parse.go

```diff
@@ -476,14 +481,10 @@ func parseKeyValueOpts(opts opts.ListOpts) ([]KeyValuePair, error) {
 
 func parseNetMode(netMode string) (NetworkMode, error) {
 	parts := strings.Split(netMode, ":")
-	switch mode := parts[0]; mode {
-	case "bridge", "none", "host":
-	case "container":
+	if parts[0] == "container" {
 		if len(parts) < 2 || parts[1] == "" {
 			return "", fmt.Errorf("invalid container format container:<name|id>")
 		}
-	default:
```

Comment:
Looks like we need to add a validation for an empty string, which this default clause was catching.